src/webui/www/public/scripts/mocha-init.js

```diff
@@ -451,6 +451,37 @@ initializeWindows = function() {
         }
     };
 
+    copyNameFN = function() {
+        var h = torrentsTable.selectedRowsIds();
+        var names = [];
+        if (h.length) {
+            var rows = torrentsTable.getFilteredAndSortedRows();
+            for (var i = 0; i < h.length; i++) {
```

Comment:
The `h.length` check is utilized in similar functions in this file, so I'm assuming (or hoping) there's some logic behind it. And the issue with using `for in` is that it loops over additional properties of h, like `$family`. To achieve the desired effect `for of` would have to be used, which is unfortunately an es6 feature.

<img width="636" alt="screen shot 2017-07-02 at 6 15 22 am" src="https://user-images.githubusercontent.com/8296030/27769003-c4adb9ac-5eee-11e7-93d5-3f68d2d4bb27.png">